index.html

```diff
@@ -2089,7 +2089,7 @@ <h2 id="docconformance">
               </p>
             </td>
           </tr>
-          <tr id="el-title" tabindex="-1">
+          <tr id="title" tabindex="-1">
```

Comment:
here's the issue.

"el-" was there to not have this conflict with the `title` attribute's `id`.

you can change the attribute's id to `attr-title` to match what you did for `contenteditable`

Reply:
Oh man! I even searched for `id="title"` in the source to see if there was a `title` attribute id in there somewhere - and there wasn't any, so I thought it was safe.

But it looks like ReSpec must be looking at the document's `<title>ARIA in HTML</title>` and adding an `h1` with `id="title"`:
```
<h1 id="title" class="title p-name">ARIA in HTML</h1>
```

Looking at a couple other W3C specs, I see they also have the `h1` with `id="title"`.

Changing ReSpec to ask them to use "doc-title" or some other id might cause breakage - i.e. maybe someone or some doc is linking into these spec titles?

I can't compete with that - I'll put it back to "el-title".

Reply:
If you prefer French over Spanish you can change it to `le-title` 😉.